MAPL_Base/CMakeLists.txt

```diff
@@ -104,9 +82,9 @@ target_link_libraries (${this} PUBLIC ${ESMF_LIBRARIES} ${MPI_Fortran_LIBRARIES}
 # in the other libaries, but this make it explicit which aspects are externally
 # used.
 file (COPY ${MAPL_SOURCE_DIR}/include/unused_dummy.H DESTINATION ${esma_include}/${this})
-file (COPY MAPL_Generic.h DESTINATION ${esma_include}/${this})
-file (COPY MAPL_Exceptions.h DESTINATION ${esma_include}/${this})
-file (COPY MAPL_ErrLog.h DESTINATION ${esma_include}/${this})
+file (COPY ${MAPL_SOURCE_DIR}/include/MAPL_Generic.h DESTINATION ${esma_include}/${this})
+file (COPY ${MAPL_SOURCE_DIR}/include/MAPL_Exceptions.h DESTINATION ${esma_include}/${this})
+file (COPY ${MAPL_SOURCE_DIR}/include/MAPL_ErrLog.h DESTINATION ${esma_include}/${this})
```

Comment:
This logic should be moved into a new CMakeLists.txt in the ./include directory

Reply:
Is this for MAPL_Base to be used independently?  I am wondering how CMakeLists.txt in the ./include would know where to copy those header files

Reply:
Sorry - I was rushing and not pulling all of the information together.   There should not be _any_ copy of these files.  (I was thinking this was the INSTALL step.)

${MAPL_SOURCE_DIR}/include   should be added to the target_include_directories with the BUILD_INTERFACE generator and <install>/include/MAPL for the INSTALL_INTERFACE